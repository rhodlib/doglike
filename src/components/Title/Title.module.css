.Title {
  margin-top: 0px;
  font-size: 3vw;
  color: var(--color-white);
  text-transform: uppercase;
  text-shadow: 0 2px 4px rgba(0, 0, 0, 0.4);
}

@media (max-width: 1080px) {
  .Title {
    font-size: 10vw;
  }
}
