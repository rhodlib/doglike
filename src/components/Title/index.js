import React from "react";
import PropTypes from "prop-types";
import styles from "./Title.module.css";

/**
 * Component that returns a h1 to use as a title.
 */
export const Title = ({ name, className = "" }) => (
  <h1 className={[styles.Title, className].join(" ")}>{name}</h1>
);

Title.propTypes = {
  name: PropTypes.string.isRequired
};

export default Title;
