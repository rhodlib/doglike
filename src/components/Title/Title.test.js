import React from "react";
import { render } from "@testing-library/react";
import Title from "./index";

it("Matches snapshot", () => {
  const { asFragment } = render(<Title name="test" />);
  expect(asFragment()).toMatchSnapshot();
});
