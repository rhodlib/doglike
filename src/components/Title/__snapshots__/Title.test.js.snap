// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches snapshot 1`] = `
<DocumentFragment>
  <h1
    class="Title "
  >
    test
  </h1>
</DocumentFragment>
`;
